.teamcity/_self/projects/WebApp.kt

```diff
@@ -1062,9 +1062,77 @@ fun e2ePreReleaseBuildType( targetDevice: String, buildUuid: String ): E2EBuildT
 		buildUuid = buildUuid,
 		buildName = "Pre-Release E2E Tests ($targetDevice)",
 		buildDescription = "Runs a pre-release suite of E2E tests against trunk on staging, intended to be run after PR merge, but before deployment to production. Will run on $targetDevice size.",
-		testGroup = "calypso-pr",
+		testGroup = "calypso-release",
+		buildParams = {
+			param("env.VIEWPORT_NAME", "$targetDevice")
+			param("env.CALYPSO_BASE_URL", "https://wpcalypso.wordpress.com")
+			param("env.ALLURE_RESULTS_PATH", "allure-results")
+		},
+		buildSteps = {
+			bashNodeScript {
+				name = "Collect Allure results"
```

Comment:
The Allure logic is related to [this PR](https://github.com/Automattic/wp-calypso/pull/69802) and copied from the PreReleaseE2ETests object.

It looks like there's a [dashboard](https://automattic.github.io/wp-calypso-test-results/) for tracking this. I'm not sure if this is still in use or not. Maybe we should ask?

On a side note, at some point, this script will run twice, once for desktop and once for mobile. Will that cause an issue for the data on the dashboard? 🤔